WcaOnRails/app/assets/javascripts/application.js

```diff
@@ -77,6 +77,14 @@ $(function() {
     }
   });
 
+  // We use dp.hide to capture clicking on a date. We then tab to the next
+  // form item, so the picker is just one click away, in case the user made
+  // a mistake or wants to select a different date. This also triggers the
+  // 'blur' event, used above to validate the date.
+  $datetimepicker.on('dp.hide', function() {
+    $('input:not(input[type=button], input[type=submit], input[type=reset]), select, textarea').eq(parseInt($('input:not(input[type=button], input[type=submit], input[type=reset]), select, textarea').index($(this)), 10) + 1).focus();
```

Comment:
What about if you're the very last element? Shouldn't this wrap around?

Listing all the types of focusable elements seems gross and not going to work in general (see how complicated it is: https://github.com/marklagendijk/jquery.tabbable/blob/master/jquery.tabbable.js#L109-L135).

Can we instead make use fo the :focusable or :tabable selectors provided by the jquery.tabbable library, and write some code that looks liek this: https://github.com/marklagendijk/jquery.tabbable/blob/master/jquery.tabbable.js#L50-L62, but instead uses `$(this)` instead of `current = $(':focus');`?